src/templates/core/category_create.html

```diff
@@ -1,44 +1,29 @@
-        <div class="u-align-center u-expanded-width-xs u-form u-form-1">
-          <form action="#" method="POST" class="u-clearfix u-form-spacing-18 u-form-vertical u-inner-form" source="custom" style="padding: 0px;">
-            <div class="u-form-group u-form-name u-form-partition-factor-2 u-form-group-1">
-              <label for="name-e4cc" class="u-form-control-hidden u-label  u-label-1" wfd-invisible="true">Name</label>
-              <input type="text" id="name-e4cc" name="name" class="u-border-1 u-border-palette-5-dark-1 u-input u-input-rectangle u-radius-5  u-input-1" required="" placeholder="dddd">
-            </div>
-            <div class="u-form-email u-form-group u-form-partition-factor-2 u-form-group-2">
-{#              <label for="email-e4cc" class="u-label  u-label-2" wfd-invisible="true">ddddd</label>#}
-              <input type="email" placeholder="Enter a valid email address" id="email-e4cc" name="email" class="u-border-1 u-border-palette-5-dark-1 u-input u-input-rectangle u-radius-5  u-input-2" required="">
-            </div>
-            <div class="u-form-email u-form-group u-form-group-3">
-              <label for="email-f97d" class="u-form-control-hidden u-label  u-label-3">Email</label>
-              <input type="email" placeholder="Enter a valid email address" id="email-f97d" name="email-1" class="u-border-1 u-border-palette-5-dark-1 u-input u-input-rectangle u-radius-5  u-input-3" required="">
-            </div>
-            <div class="u-form-group u-form-phone u-form-group-4">
-              <label for="phone-810d" class="u-form-control-hidden u-label  u-label-4"></label>
-              <input type="tel" pattern="\+?\d{0,3}[\s\(\-]?([0-9]{2,3})[\s\)\-]?([\s\-]?)([0-9]{3})[\s\-]?([0-9]{2})[\s\-]?([0-9]{2})" placeholder="Enter your phone (e.g. +14155552675)" id="phone-810d" name="phone" class="u-border-1 u-border-palette-5-dark-1 u-input u-input-rectangle u-radius-5  u-input-4" required="">
-            </div>
-            <div class="u-form-group u-form-message u-form-group-5">
-              <label for="message-e4cc" class="u-form-control-hidden u-label  u-label-5" wfd-invisible="true">Message</label>
-              <textarea placeholder="Enter your message" rows="4" cols="50" id="message-e4cc" name="message" class="u-border-1 u-border-palette-5-dark-1 u-input u-input-rectangle u-radius-5  u-input-5" required=""></textarea>
+{% load static %}
+{% load widget_tweaks %}
+{% block head_title %}{% trans "Add Category" %}
+{% endblock %}
+{% block card %}
+    <div class="col-lg-12 d-flex justify-content-center mt-3">
+        <h1>{% trans "Add Category" %}</h1>
+    </div>
+ <form action="{% url 'core:category-create' %}" method="post" class="card-body" id="profileUpdateForm">
+        {% csrf_token %}
+        {% for field in form %}
+            <div class="row mb-3 justify-content-center">
+                <div class="col-sm-9 text-secondary">
+                    {{ field }}
```

Comment:
Instead iteration thorough form, you should use {{form.field_name }} or {% render_field form.field_name ...another params %}

Reply:
done